app/src/main/java/com/kickstarter/ui/viewholders/CreatorDashboardHeaderViewHolder.java

```diff
@@ -0,0 +1,131 @@
+package com.kickstarter.ui.viewholders;
+
+
+import android.content.Intent;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.util.Pair;
+import android.view.View;
+import android.widget.TextView;
+
+import com.kickstarter.R;
+import com.kickstarter.libs.KSCurrency;
+import com.kickstarter.libs.KSString;
+import com.kickstarter.libs.RefTag;
+import com.kickstarter.libs.utils.ProjectUtils;
+import com.kickstarter.models.Project;
+import com.kickstarter.models.ProjectStats;
+import com.kickstarter.ui.IntentKey;
+import com.kickstarter.ui.activities.ProjectActivity;
+import com.kickstarter.viewmodels.CreatorDashboardHeaderHolderViewModel;
+
+import java.math.RoundingMode;
+
+import butterknife.Bind;
+import butterknife.BindString;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+
+import static com.kickstarter.libs.rx.transformers.Transformers.observeForUI;
+import static com.kickstarter.libs.utils.ObjectUtils.requireNonNull;
+
+public final class CreatorDashboardHeaderViewHolder extends KSViewHolder {
+
+  private final CreatorDashboardHeaderHolderViewModel.ViewModel viewModel;
+
+  protected @Bind(R.id.creator_dashboard_amount_raised) TextView amountRaisedTextView;
+  protected @Bind(R.id.creator_dashboard_backer_count) TextView backerCountTextView;
+  protected @Bind(R.id.creator_dashboard_funding_text) TextView fundingTextTextView;
+  protected @Bind(R.id.creator_dashboard_percent) TextView percentTextView;
+  protected @Bind(R.id.creator_dashboard_project_name) TextView projectNameTextView;
+  protected @Bind(R.id.creator_dashboard_time_remaining) TextView timeRemainingTextView;
+  protected @Bind(R.id.creator_dashboard_time_remaining_text) TextView timeRemainingTextTextView;
+  protected @BindString(R.string.discovery_baseball_card_stats_pledged_of_goal) String pledgedOfGoalString;
+
+  private KSString ksString;
+  private KSCurrency ksCurrency;
+
+  public CreatorDashboardHeaderViewHolder(final @NonNull View view) {
+    super(view);
+
+    viewModel = new CreatorDashboardHeaderHolderViewModel.ViewModel(environment());
+    ButterKnife.bind(this, view);
+
+    this.ksCurrency = this.environment().ksCurrency();
+    this.ksString = this.environment().ksString();
+
+    viewModel.outputs.currentProject()
```

Comment:
I wonder if some of this code needs to be updated or deleted? We're subscribing to the project rather than `timeRemainingText`, seems like we're not using the correct output. Same applies for the pledged of goal output below.